cfgmgr/buffermgrdyn.cpp

```diff
@@ -1892,10 +1900,16 @@ task_process_status BufferMgrDynamic::handleBufferMaxParam(KeyOpFieldsValuesTupl
 task_process_status BufferMgrDynamic::handleDefaultLossLessBufferParam(KeyOpFieldsValuesTuple &tuple)
 {
     string op = kfvOp(tuple);
-    string newRatio = "0";
+    string newRatio = "";
 
     if (op == SET_COMMAND)
     {
+        if (m_bufferPoolLookup.find(INGRESS_LOSSLESS_PG_POOL_NAME) == m_bufferPoolLookup.end())
+        {
+            SWSS_LOG_NOTICE("%s has not been configured, need to retry", INGRESS_LOSSLESS_PG_POOL_NAME);
```

Comment:
Suggest not to spew continuous logs in retry logic. Can you please change to INFO?

Reply:
Fixed. Also fixed most of the other `NOTICE` log in retry logic.